dags/bqetl_artifact_deployment.py

```diff
@@ -95,11 +95,25 @@
             "script/publish_public_data_views --target-project=moz-fx-data-shared-prod && "
             "script/publish_public_data_views --target-project=mozdata"
         ],
-        docker_image=docker_image,
+        image=docker_image,
         get_logs=False,
         trigger_rule=TriggerRule.ALL_DONE,
     )
 
+    publish_metadata = GKEPodOperator(
+        task_id="publish_metadata",
+        cmds=["bash", "-x", "-c"],
+        arguments=[
+            "script/bqetl generate all --use-cloud-function=false && "
+            "script/bqetl metadata publish moz-fx-data-shared-prod.* --project_id=moz-fx-data-shared-prod && "
+            "script/bqetl metadata publish mozdata.* --project_id=mozdata && "
+            "script/bqetl metadata publish moz-fx-data-marketing-prod.* --project_id=moz-fx-data-marketing-prod && "
+            "script/bqetl metadata publish moz-fx-data-experiments.* --project_id=moz-fx-data-experiments"
```

Comment:
```suggestion
            "script/bqetl generate all --use-cloud-function=false && "
            "script/bqetl metadata publish '*' --project_id=moz-fx-data-shared-prod && "
            "script/bqetl metadata publish '*' --project_id=mozdata && "
            "script/bqetl metadata publish '*' --project_id=moz-fx-data-marketing-prod && "
            "script/bqetl metadata publish '*' --project_id=moz-fx-data-experiments"
```

`'` needed when using wildcards. `project_id` is passed into `paths_matching_name_pattern` and used when looking for matching files, so we can simplify this a ltittle here.